update.cf

```diff
@@ -131,9 +131,9 @@ bundle agent cfengine_internal_standalone_self_upgrade_execution
 
     trigger_upgrade.!at_desired_version::
 
-      "$(exec_prefix)$(sys.cf_agent)"
+      '$(exec_prefix)$(sys.cf_agent) --inform --timestamp --file "$(this.promise_dirname)$(const.dirsep)standalone_self_upgrade.cf" --define trigger_upgrade,update_cf_initiated >"$(sys.statedir)/standalone_self_upgrade_$(sys.cdate).log"'
```

Comment:
@nickanderson so I guess we can replace `$(sys.statedir)` with `$(sys.workdir)$(const.dirsep)outputs` then?